app/assets/javascripts/atomic_cms.js

```diff
@@ -3,7 +3,7 @@
 
 (function() {
   'use strict';
-  var findElementIndex, page, positionEditBox, scrollTo, cmsType;
```

Comment:
I'm pretty sure @bbrock25 added cmsType so that editable pages don't have to be stored on an instance of the `Page` model.